app/client/keybase/README.md

```diff
@@ -52,21 +52,32 @@ Keys can be created without a password by specifying an empty (`""`) passphrase.
 
 ### Keybase Code Structure
 
-```bash
-app
-└── client
-    └── keybase
-          ├── README.md
-          ├── keybase.go
-          ├── keybase_test.go
-          └── keystore.go
+```sh
+$ tree app/client/keybase/
```

Comment:
Just a NIT - I don't think we need this line the output is really what we are trying to show

Reply:
I compare NITs to vegetables, I hate vegetables 🤣 but i still eat em.